modules/social_features/social_event/config/install/field.field.node.event.field_hide_enrollments.yml

```diff
@@ -0,0 +1,22 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_hide_enrollments
+    - node.type.event
+id: node.event.field_hide_enrollments
+field_name: field_hide_enrollments
+entity_type: node
+bundle: event
+label: 'Hide enrollments'
+description: 'Allows to restrict access to enrollments for users'
```

Comment:
@nechai Did you confirm this copy with the Design team?

Reply:
Confirmed with Design team